nomad/structs/structs.go

```diff
@@ -2325,9 +2329,13 @@ func (n *Node) HasEvent(msg string) bool {
 
 // Stub returns a summarized version of the node
 func (n *Node) Stub(fields *NodeStubFields) *NodeListStub {
-
+	var maxAllocs uint64
```

Comment:
I'm not sure that this needs to be a uint but I made it one because the other integers in the NodeListStub were 🤷‍♀️